lms/djangoapps/ccx/migrations/0003_add_master_course_staff_in_ccx.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from ccx_keys.locator import CCXLocator
+from courseware.courses import get_course_by_id
+from contextlib import contextmanager
+from django.db import migrations
+
+from lms.djangoapps.ccx.utils import (
+    add_master_course_staff_to_ccx,
+    remove_master_course_staff_from_ccx,
+)
+
+
+@contextmanager
```

Comment:
It looks like ccx_course is also a lift from utils.py, back from when the `with ccx_course(ccx_key) as course_ccx` existed in the migration, so that should also go away.  You can also clean up the unused import.